styles.css

```diff
@@ -0,0 +1,101 @@
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+.container {
+  margin: 0 auto;
+  padding: 20px;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: #f4f5f7;
+}
+
+img {
+  width: 100%;
+}
+
+.bx {
+  font-size: 35px;
+}
+
+p {
+  margin: 10px 0;
+}
+
+.logo {
+  font-size: 20px;
+  font-weight: 600;
+  color: #6070ff;
+}
+
+.navbar i {
+  color: #6070ff;
+}
+
+.navbar {
+  background-color: #fff;
+  position: fixed;
+  z-index: 1;
+  width: 100%;
+  height: 70px;
+  color: #6070ff;
+}
+
+.navbar-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-transform: none;
+}
+
+.navbar ul {
+  display: none;
+}
+
+.hero {
+  height: 720px;
+  top: 70px;
+  position: relative;
+  background-image: url(images/bg/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-position: center 50px;
+  border-bottom-left-radius: 120px;
+  background-color: #fff;
+  width: 100%;
+}
+
+.hero-text {
+  margin-top: 120px;
+}
```

Comment:
- [ ] Please consider using `Flexbox` to align the headline contents in this code block. By utilizing `Flexbox`, you can achieve a responsive layout that adapts well to different screen sizes. Flexbox provides powerful alignment and distribution options, allowing you to easily control the positioning of the headline elements. This will enhance the visual presentation and ensure a consistent user experience across devices. Feel free to explore flexbox properties such as `display: flex`, `justify-content`, and `align-items` to achieve the desired alignment.